src/components/Comments/Comment.js

```diff
@@ -273,7 +273,7 @@ const Comment = ({
           </ButtonHightlight>
         </TouchableOpacity>
 
-        {totalVote != 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Nice work on changing the comparison operator from `!=` to `!==`. It's always better to use strict equality (`===`) or inequality (`!==`) operators in JavaScript as they consider both value and type, which makes your code more predictable and less prone to bugs.


<!-- This is an auto-generated comment by OSS CodeRabbit -->